it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionControllerISpec.scala

```diff
@@ -152,32 +154,22 @@ trait SubscriptionControllerISpec extends BaseISpec with SessionDataMissingSpec
     }
 
     "display the static page content" in new RequestWithSessionDetails {
-      resultOf(request) should containMessages(
+
+      val result = resultOf(request)
+
+      result should containMessages(
         "subscriptionComplete.title",
         "subscriptionComplete.h1",
         "subscriptionComplete.accountName",
         "subscriptionComplete.h2",
         "subscriptionComplete.bullet-list.1",
-        "subscriptionComplete.bullet-list.2"
+        "subscriptionComplete.bullet-list.2",
+        "subscriptionComplete.bullet-list.3",
+        "subscriptionComplete.p2",
+        "subscriptionComplete.p3",
+        "subscriptionComplete.p4"
       )
-
-      bodyOf(resultOf(request)) should include(hasMessage("subscriptionComplete.p1", "AARN-000-0001"))
-      bodyOf(resultOf(request)) should include(hasMessage("subscriptionComplete.p2", "https://www.gov.uk/guidance/get-an-hmrc-agent-services-account"))
-    }
-
-    "contain a button to continue journey" when {
-      "a continue URL exists in the session, show a generic 'Continue' button using that URL" in new RequestWithSessionDetails {
-        val continueUrl = ContinueUrl("/test-continue-url")
-        sessionStoreService.currentSession.continueUrl = Some(continueUrl)
-
-        resultOf(request) should containSubstrings(s">${htmlEscapedMessage("subscriptionComplete.button.continueJourney")}</a>", continueUrl.url)
-      }
-
-      "no continue URL exists in the session, show a button with a link in AS services" in new RequestWithSessionDetails {
```

Comment:
don't we need the above tests ?

Reply:
Those elements have been removed from the page so no we don't need the tests